scripts/auto-rebase/assets.yaml

```diff
@@ -208,3 +208,43 @@ assets:
 
   - file: embed.go
     ignore: "it's a MicroShift specific Go file that embeds into a binary"
+
+  - dir: components/operator-lifecycle-manager/
```

Comment:
I think we don't want these in the `assets/components/`. That dir is being embedded into the microshift binary and since these manifests are meant to be delivered via rpm it's unnecessary

Reply:
Ok! What other path do you recommend me to use?

Reply:
Updated to use optional/ path.